File: src/Example3Java/EX16.java
package Example3Java;

public class EX16 {
    public static void main(String[] args) {
        int[] x = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
        int[] y = new int[x.length * 2];
        for (int i = 0; i < x.length; i++) {
            y[i] = x[i];
        }
    }
}

File: src/Example3Kotlin/EX16.kt
object EX16 {
    @JvmStatic fun main(args: Array<String>) {
        val x = listOf(1, 2, 3, 4, 5, 6, 7, 8, 9)
        var y = IntArray(x.size * 2)
        for (i in x.indices) y[i] = x[i]
        println(y.contentToString())
    }
}
